### ISSUE TITLE
About Special Types in Generics

### ISSUE THREAD

**Issue by baykus871** [h3rald/min#98]
Generics doesn't accept special types in. They only accept native types: `int`, `bool` etc.
However i want to use my special typo in generics of operator signatures.
For example:
```
(char|num :t)...
```
It throws an error:
```
(!) src/utils.min(73,4) [::]: Invalid type 'char|num' in generic in signature at position 1
    src/utils.min(73,4) in symbol: operator
    src/utils.min(73,4) in symbol: ::
    src\base.min(1,18) in symbol: require
```
???????

**Comment by h3rald** [comment]
OK, this is actually a bug! But luckily easy to fix. Will provide a fix soon!

**Comment by baykus871** [comment]
I will be waiting your fixed release. ⏳  👀

**Comment by h3rald** [comment]
This should be fixed in v0.29.1, please reopen if it still doesn't work!
